test/integration/migrations_test.rb

```diff
@@ -360,7 +370,7 @@
   describe "PostgreSQL" do
     before do
       @database  = "#{ db_prefix }_#{ random_token }"
-      @uri = uri = "postgres://#{ POSTGRES_USER }@localhost/#{ @database }"
+      @uri       = uri = "#{ adapter_prefix }postgresql://127.0.0.1/#{ @database }?user=#{ POSTGRES_USER }"
```

Comment:
This is a tricky one, JDBC does not support this format :/ But Ruby adapters supports those `?user=user123` one. Do you think it's a problem to use this format?